jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/layout.xsd

```diff
@@ -3573,6 +3573,7 @@
 
                 <xs:attribute name="dataLoader" type="xs:string" use="required"/>
                 <xs:attribute name="autoApply" type="xs:boolean"/>
+                <xs:attribute name="applyShortcut" type="xs:string"/>
```

Comment:
Do we have XML loading test for this attribute?

Reply:
Will be added with https://github.com/jmix-framework/jmix/issues/2117